stubs/TgCrypto/tgcrypto/__init__.pyi

```diff
@@ -0,0 +1,6 @@
+def ige256_encrypt(data: bytes, key: bytes, iv: bytes) -> bytes: ...
+def ige256_decrypt(data: bytes, key: bytes, iv: bytes) -> bytes: ...
+def ctr256_encrypt(data: bytes, key: bytes, iv: bytes, state: bytes) -> bytes: ...
+def ctr256_decrypt(data: bytes, key: bytes, iv: bytes, state: bytes) -> bytes: ...
+def cbc256_encrypt(data: bytes, key: bytes, iv: bytes) -> bytes: ...
+def cbc256_decrypt(data: bytes, key: bytes, iv: bytes) -> bytes: ...
```

Comment:
Thanks!

These functions use `PyArg_ParseTuple`, so they only accept positional-only arguments. Also, using `y*` means that they accept any buffer object:

```suggestion
from typing_extensions import Buffer

def ige256_encrypt(data: Buffer, key: Buffer, iv: Buffer, /) -> bytes: ...
def ige256_decrypt(data: Buffer, key: Buffer, iv: Buffer, /) -> bytes: ...
def ctr256_encrypt(data: Buffer, key: Buffer, iv: Buffer, state: Buffer, /) -> bytes: ...
def ctr256_decrypt(data: Buffer, key: Buffer, iv: Buffer, state: Buffer, /) -> bytes: ...
def cbc256_encrypt(data: Buffer, key: Buffer, iv: Buffer, /) -> bytes: ...
def cbc256_decrypt(data: Buffer, key: Buffer, iv: Buffer, /) -> bytes: ...
```